docs/4.3/architecture/teleport_auth.md

```diff
@@ -224,9 +224,9 @@ back-ends as shown in the table below:
 
 Data Type        | Supported Back-ends       | Notes
 -----------------|---------------------------|---------
-Cluster state    | `dir`, `etcd`, `dynamodb` | Multi-server (HA) configuration is only supported using `etcd` and `dynamodb` back-ends.
-Audit Log Events | `dir`, `dynamodb`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
-Recorded Sessions| `dir`, `s3`               | `s3` is mandatory if `dynamodb` is used for the audit log.
+Cluster state    | `dir`, `etcd`, `dynamodb`,`firestore` | Multi-server (HA) configuration is only supported using `etcd`, `dynamodb`, and `firestore` back-ends.
+Audit Log Events | `dir`, `dynamodb`, `firestore`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
+Recorded Sessions| `dir`, `s3`, `gcp`               | `s3` is mandatory if `dynamodb` is used for the audit log.
```

Comment:
`gcp` isn't a valid entry here. I looked into it, and nothing in storage is set, it looks like this. `gs://` for Google Cloud Storage is required. 

```diff
  storage:
    type: firestore
    collection_name: cluster-data
    # Credentials: Path to google service account file, used for Firestore and Google Storage.
    credentials_path: /var/lib/teleport/gcs_creds.json
    project_id: example_Teleport-Project-Name
    audit_events_uri: 'firestore://events?projectID=example_Teleport-Project-Name&credentialsPath=/var/lib/teleport/gcs_creds'
+    audit_sessions_uri: 'gs://teleport-session-storage-2?`
```

Reply:
also note this bug https://github.com/gravitational/teleport/issues/3992

Reply:
@benarent  I changed it to gs and put a note that gs is google storage.  Anything else we want to put here?

Reply:
`gs` isn't currently supported, that's a bug. It can only be set via the audit_sessions_uri: 'gs:// for now.